### PR TITLE
Added minimal window size

### LINKED ISSUE #8: Minimal window size is too small
screenshot

### PR DESCRIPTION
**Pull request by ondralukes**
Fixed #8 
Minimal size is now 100x200

### CODE REVIEW

**Comment by humdingerb** [comment]
There's a much easier way. Search the use of the function SetSizeLimits() and adjust it. See the BeBook for its description.

**Comment by ondralukes** [comment]
`SetSizeLimits()` already was in `CDockApp::ReadyToRun()`, I haven't noticed it.

File: src/DockApp.cpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ void CDockApp::ReadyToRun()
 
 	mwDock->MoveTo( x, y );
 	mwDock->ResizeTo( xw, yw );
-	mwDock->SetSizeLimits( 48, 200, 100, r.bottom - 4 ); 
+	mwDock->SetSizeLimits( 100, 200, 200, r.bottom - 4 );
```
Comment (humdingerb) [inline src/DockApp.cpp]: I don't think there's a valid reason to limit the max width to 200px. If the user wants to make it wider, why not?
Just set it to 10000, I'd say.

Comment (ondralukes) [reply src/DockApp.cpp]: OK, wait a minute.

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**